main.js

```diff
@@ -0,0 +1,90 @@
+
+// Task #1
+
+// document.querySelector('#first-btn').onclick = transformToСentury;

```

Comment:
Try to use a common event handler `addEventListener`
The onclick is just a property. Like all object properties, if we write more than one property, it will be overwritten.